source/framework/vx_graph_verify.c

```diff
@@ -104,7 +104,7 @@ static vx_status ownGraphUpdateImageRefAfterKernetInit(vx_graph graph, vx_image
 static vx_status ownGraphUpdateObjArrRefAfterKernetInit(vx_graph graph, vx_object_array exemplar, vx_object_array ref);
 static vx_status ownGraphUpdatePyramidRefAfterKernetInit(vx_graph graph, vx_pyramid exemplar, vx_pyramid ref);
 
-/* Add's data reference to a list, increments number of times it is refered as input node */
+/* Add's data reference to a list, increments number of times it is referred as input node */
```

Comment:
I believe this should have been taken care of in the other commits as well

Reply:
to be removed from the PR @RB